app/models/indexing/index_queue.rb

```diff
@@ -12,6 +12,7 @@ def self.all
   end
 
   def self.get_key(klass, label)
+    klass = klass.is_a?(Class) ? klass.base_class : klass
```

Comment:
What's the reason behind that change I'm not sure I understand?

Reply:
There was this [discussion](https://github.com/otwcode/otwarchive/pull/4230#issuecomment-1151538913) in the past PR.